apps/rush/src/RushCommandSelector.ts

```diff
@@ -45,12 +45,14 @@ export class RushCommandSelector {
       }
       Rush.launchRushX(launcherVersion, {
```

Comment:
Is there a reason not to just directly pass the options object?

Reply:
Seems safe to me, since (1) the interfaces are the same type from the same version of the library, and (2) the `options` object is privately created by this project.